test/salt/states/collectd-deps.sls

```diff
@@ -0,0 +1,10 @@
+{% set postgresql_plugin = salt['grains.filter_by']({
```

Comment:
Since the relevant package is required for working with the PostgreSQL plugin, then this state should be added to `collectd/postgresql.sls` instead.  The package names can be handled in `map.jinja`.

We only use `test/salt/states/...` when a state is needed purely for testing/CI only.

Reply:
I had the reverse idea: that is not the job of this formula to manage every dependency for every plugin, or at least it doesn't now.

Reply:
The state was moved to `map.jinja` and called in `postgresql.sls` state. Rebase done and everything seems ok https://gitlab.com/saltstack-formulas/collectd-formula/-/pipelines/544095260